plugins/modules/power_system.py

```diff
@@ -110,8 +110,9 @@
             - C(modify_hwres) Makes hardware resource configurations of specified I(system_name).
             - C(enable_pcm) Enables the Performance and Capacity Monitoring for specified types of utilization data.
             - C(disable_pcm) Disables the Performance and Capacity Monitoring for specified types of utilization data.
+            - C(list_pcm) Provides the status of the Performance and Capacity Monitoring of the managed system
```

Comment:
please provide more details of what details it shows

Reply:
Done